monitoring/mongodb/alerts.test.yaml

```diff
@@ -333,12 +333,16 @@ tests:
   - name: MongoDbRSNotSynced
     interval: 1m
     input_series:
-      - series: mongodb_rs_members_state{namespace="zenko", rs_nm="rs0", pod="data-db-mongodb-sharded-mongos-0", member_state="SECONDARY"}
-        values: 0x10
-      - series: mongodb_rs_members_state{namespace="zenko", rs_nm="rs0", pod="data-db-mongodb-sharded-mongos-1", member_state="SECONDARY"}
-        values: 0x10
-      - series: mongodb_rs_members_state{namespace="zenko", rs_nm="rs0", pod="data-db-mongodb-sharded-mongos-2", member_state="SECONDARY"}
-        values: 1x10
+      - series: mongodb_rs_members_state{namespace="zenko", rs_nm="rs0", pod="data-db-mongodb-sharded-mongos-0", member_state="SECONDARY", instance="10.233.226.137:9216"}
+        values: 2x10
+      - series: mongodb_rs_members_state{namespace="zenko", rs_nm="rs0", pod="data-db-mongodb-sharded-mongos-1", member_state="SECONDARY", instance="10.233.226.138:9216"}
+        values: 2x10
+      - series: mongodb_rs_members_state{namespace="zenko", rs_nm="rs0", pod="data-db-mongodb-sharded-mongos-2", member_state="SECONDARY", instance="10.233.226.139:9216"}
+        values: 2x10
+      - series: mongodb_rs_members_state{namespace="zenko", rs_nm="rs1", pod="data-db-mongodb-sharded-mongos-0", member_state="SECONDARY", instance="10.233.226.140:9216"}
+        values: 2x10
+      - series: mongodb_rs_members_state{namespace="zenko", rs_nm="rs1", pod="data-db-mongodb-sharded-mongos-1", member_state="SECONDARY", instance="10.233.226.140:9216"}
```

Comment:
`instance` is the IP of the pod which exposes the metric: so multiple `pod` should not have the same value for `instance`, right?

Reply:
yes , this is why we have an example with multiple IP and one with the same one ( the first fires alerts while the second doesn't )

Reply:
it should not just be an "example" but also match reality : we should have metrics which are similar to what is in a real cluster, and then introduce changes to simulate various error conditions...